### DISCUSSION TITLE
view continuous

### DISCUSSION THREAD

**Discussion by mcanonic** [linuxmint/discussions#515] [Ideas]
Is it possible make "view -> continuous" set by default?
Thanks

**Comment by spamegg1** [comment]
Are you talking about the Xreader Document Viewer?

**Reply by spamegg1** [reply]
There is a "Save current settings as default" option under `Edit`.

**Reply by saberworks** [reply]
~~This doesn't seem to work for me.  I'm on mint 21.3 and xreader 4.0.2.  If I change to continuous view, click the edit -> save current settings ad default, then exit the program, and re-launch, it's back to page view (where mouse scrolling won't go to the next page).~~

After doing it multiple times on two different documents, it finally stuck.
